eng/pipelines/coreclr/templates/build-perf-maui-apps.yml

```diff
@@ -178,6 +178,9 @@ steps:
       cp MauiTesting.csproj MauiTesting.csproj.bak
       sed -i'' -e 's/net6.0-ios;net6.0-maccatalyst/net6.0-ios/g' MauiTesting.csproj
 
+      # Test xharness with the app id set to net.dot.mauitesting. This is necessary for the iOS on device signing
+      sed -i'' -e 's/com.companyname.mauitesting/net.dot.mauitesting/g' MauiTesting.csproj
```

Comment:
instead of using `sed` I think in this case we could override the ID on the msbuild command line via `/p:ApplicationId=net.dot.mauitesting`

Reply:
I am assuming you are talking about during the dotnet publish step? Let me try that out.